README.md

```diff
@@ -66,11 +72,18 @@ Please see [Azure IoT Edge Dev Resources](https://github.com/jonbgallant/azure-i
     - Make sure you enable Admin Access when you create the Azure Container Registry
 
 #### Automated Setup
+
+You can use the **Azure IoT Edge Dev Tool** to create a new IoT Hub and a new Edge device. This command will also print the corresponding connection strings 
+```
+iotedgedev azure --setup
+```
     
-You can also deploy the IoT Hub and Container Registry with this **Deploy to Azure** template:
+Alternatively, you can deploy the IoT Hub **and** Container Registry with this **Deploy to Azure** template:
 
 [![Azure Deployment](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fjonbgallant%2Fazure-iot-edge-dev-tool%2Fmaster%2Fassets%2Fdeploy%2FARMDeployment%2Fazuredeploy.json)
 
+> Note: If you do not need a Container Registry, or planning to use your local registry, running the **iotedgedev azure --setup** command will provision all required Azure compontents
```

Comment:
components

Note: If you do not need a Container Registry, or are planning to use a local registry, then you should run the **iotedgedev azure --setup** command instead of running this "Deploy to Azure" template, because the template includes a Container Registry.